### DISCUSSION TITLE
Downloading Student Roster No Longer Functioning

### DISCUSSION THREAD

**Discussion by chrissphinx** [community/community#61665] [GitHub Education]
### Hi everyone,

Seemingly another UX change that has broken something that used to work. Now when clicking on the "Download" button when viewing the current roster of a course, a modal dialog is shown with a "Download roster" button. Clicking this button has no effect. Nothing is downloaded. Incredibly annoying. Please make it so that we can download student roster .csv files again.

**Comment by caburu** [comment]
Same here :( #61148
